modules/ting_field_search/ting_field_search.ting_field_search_default_profiles.inc

```diff
@@ -927,6 +957,43 @@ function ting_field_search_ting_field_search_default_profiles() {
   );
   $export['spil'] = $profile;
 
+  $profile = new stdClass();
+  $profile->disabled = FALSE; /* Edit this to true to make a default profile disabled initially */
+  $profile->api_version = 1;
+  $profile->name = 'tegneserier';
+  $profile->title = 'Tegneserier';
+  $profile->config = array(
+    'search_request' => array(
+      'agency' => '',
+      'well_profile' => '',
+      'query' => 'term.type=tegneserie or term.type="graphic novel"',
+      'new_materials' => '',
+      'allow_empty' => 0,
+    ),
+    'user_interaction' => array(
+      'exposed' => 1,
+      'sticky' => 1,
+      'alter_links' => 1,
+      'grouping' => 'default',
+      'visibility' => 'all',
+      'pages' => '',
+    ),
+    'search_result' => array(
+      'results_per_page' => '10',
+      'default_sort' => 'date_descending',
+      'ranking' => 'rank_frequency',
+    ),
+    'facets' => array(
+      'use_facets' => 0,
+      'number_of_facets' => 25,
+      'facet_count' => 5,
+      'term_count' => 5,
+    ),
+  );
+  $profile->weight = 11;
+  $profile->facets = array();
+  $export['tegneserier'] = $profile;
```

Comment:
Is it intentional that this profile has been added?

Reply:
It's back in feb. so i really don't remember who request this!